ebos/collecttoiorank.cc

```diff
@@ -898,5 +903,10 @@ template class CollectDataToIORank<Dune::CpGrid,
 template class CollectDataToIORank<Dune::PolyhedralGrid<3,3,double>,
                                    Dune::PolyhedralGrid<3,3,double>,
                                    Dune::GridView<Dune::PolyhedralGridViewTraits<3,3,double,Dune::PartitionIteratorType(4)>>>;
+#if HAVE_DUNE_FEM
```

Comment:
afaik this will override in general, so it should probably be an #else for the other ?

Reply:
Better now?

Reply:
aye